_includes/setup/path-win.md

```diff
@@ -0,0 +1,24 @@
+### Update your path
+
+You can update your PATH variable for the current session only at the command line,
+as shown in [Clone the Flutter repo](./#clone-the-repo). You'll probably want to 
```

Comment:
OK, will remove for now. Are we monitoring feedback channels for Windows users, so if there's an issue w/some common configuration, we'll know?